core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.controller.ts

```diff
@@ -174,38 +186,27 @@ angular.module('oppia').directive('topicsAndSkillsDashboardPage', [
             ctrl.activeTab = tabName;
             if (ctrl.activeTab === ctrl.TAB_NAME_TOPICS) {
               ctrl.goToPageNumber(ctrl.topicPageNumber);
-            } else if (ctrl.activeTab === ctrl.TAB_NAME_UNTRIAGED_SKILLS) {
-              ctrl.goToPageNumber(ctrl.skillPageNumber);
+            } else if (ctrl.activeTab === ctrl.TAB_NAME_SKILLS) {
+              ctrl.initSkillDashboard();
             }
           };
 
+          ctrl.initSkillDashboard = function() {
+            ctrl.skillStatusOptions = [];
+            ctrl.more = true;
```

Comment:
'more' seems too generic for a name. What does it mean in this context?

Reply:
Yeah right, done.